packages/vscode-extension/src/controls/QuickStart.tsx

```diff
@@ -270,7 +275,7 @@ export default class QuickStart extends React.Component<any, any> {
                   className="capabilitiesVideo"
                   controls
                   disablePictureInPicture
-                  onPlay={this.onWatchVideo}
+                  onPlay={this.onVideoPlay}
```

Comment:
To call onWatchVideo() here will send telemetry twice, so separate play action with following actions